app/models/school.rb

```diff
@@ -32,4 +32,23 @@ def sierra_code
 
     return match.sierra_code
   end
+
+  
+  # Get all active schools
+  def self.active_schools
+    active.map do |school|
+      [school.name + school_code(school), school.id] 
+    end
+  end
+
+
+  def self.inactive_school(school)
```

Comment:
This would change if you like the suggestion below. Then there would be one instance method called, say, `label_and_id` or `label_id_array`, and that method would handle active and inactive schools appropriately.